opt-tools/docker/images/app/Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM openjdk:11-jre-slim
-CMD cd ../../../../; mvn package;
+WORKDIR /
+CMD mvn package
```

Comment:
OK, this splitting looks great, but it won't work as expected because you have no project files in the image - you either might need: 1) to copy your project inside an image AND after that start to `mvn package`; 2) or, mount your local directory to the container after build part and work with files after container up and running via docker-compose's `command` for example.